content/en/events/isc-2025.md

```diff
@@ -63,6 +63,403 @@ Whether you’re joining in person or virtually, InnerSource Summit 2025 promise
+</div>
+
+<div class="clearfix speaker-bio">
+<img alt="Carlos Navarro photo" src="/images/events/speakers/Summit 2025/Carlos Navarro.jpeg" width="200" class="speaker-photo" id="Carlos_Navarro" />
+
+
+**Carlos Navarro** (Building InnerSource Foundations Through InnerSource Patterns)
+
+
+Carlos Navarro Segarra is a Principal Engineer at AXA with deep expertise in Java development and architecture. His journey spans from Java Developer to Lead Developer and Solutions Architect, where he focused on scalable systems and system performance. Now, as Principal Engineer and Solutions Architect Lead, he drives developer productivity and operational efficiency through DevOps practices and Platform Engineering, while overseeing the OSPO.
+</div>
+
+<div class="clearfix speaker-bio">
+<img alt="Caroline T Jones photo" src="/images/events/speakers/Summit 2025/Caroline T Jones.jpeg" width="200" class="speaker-photo" id="Caroline+T_Jones" />
+
+
+**Caroline T Jones** (Kickstart with a Contribfest)
+
+
+Caroline Jones is an Engineering Manager at The Aerospace Corporation leading a team developing cloud native solutions to tackle some of the most difficult problems in space. She also leads Aerospace’s initiative for software standards and developer best practices, including spearheading the InnerSource project. Caroline began her career at Aerospace as an intern in 2016 before joining full-time in 2017 after completing her B.S. in Computer Engineering at Boston University. She resides in Santa Monica, CA, and has far too many hobbies to list here. 
+</div>
+
+
+<div class="clearfix speaker-bio">
+<img alt="Chamindra de Silva photo" src="/images/events/speakers/Summit 2025/Chamindra de Silva.jpeg" width="200" class="speaker-photo" id="Chamindra_de_Silva" />
+
+
+**Chamindra de Silva** (InnerSource Value Metrics)
+
+
+Chamindra de Silva is long time Open Source advocate and contributor originally from Sri Lanka and was active promoting Free and Open Source with FSF and OSI. He has contributions to Apache, Google Summer of Code, UNDP IOSN networks. Open Source projects he has lead have been awarded from SourceForge and Free Software Foundation in the past particularly for his work in the Humanitarian Open Source Domain. He is presently working in Citi in London being the InnerSource Project Manager and Solution Architect for some of the leading InnerSource projects in the organization and is member of the InnerSource governance program. He has published articles and papers in CACM, IEEE, IDRC, UNDP, UN ESCAP and CMI. ( I will be presenting with Daniel Izquierdo Cortázar)
+</div>
+
+<div class="clearfix speaker-bio">
+<img alt="Clare Dillon photo" src="/images/events/speakers/Summit 2025/Clare Dillon.jpeg" width="200" class="speaker-photo" id="Clare_Dillon" />
```

Comment:
This photo doesn't exist, yet, so it looks broken when the website is trying to display.

<img width="1366" height="768" alt="Screenshot 2025-09-25 2 15 35 PM" src="https://github.com/user-attachments/assets/e3d2d307-3664-44fd-b746-e075c65968a9" />

Need to just have no photo (e.g. white square) or silhouette photo until you get the real ones.